content/en/examples/secret/tls-auth-secret.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: v1
+kind: Secret
+metadata:
+  name: secret-tls
+type: kubernetes.io/tls
+data:
+  # the data is abbreviated in this example
+  tls.crt: |
+    MIIC2DCCAcCgAwIBAgIBATANBgkqh ...
+  tls.key: |
+    MIIEpgIBAAKCAQEA7yn3bRHQ5FHMQ ...
```

Comment:
Is this manifest `apply`-able? If not, I'd prefer to change this so that the data are at least base64 decodable (even if not an actual valid certificate and key pair)

Also, that data is wrong; I can tell that this isn't PEM format just from familiarity with DER and X.509.

Reply:
Updated with the certificate contents of www.example.com and a base64 encoded string as the key